gluu-server/opt/gluu/jetty/oxauth/custom/scripts/sic/oob.py

```diff
@@ -97,12 +98,13 @@ def AuthenticateOutOfBand(self, requestParameters):
         authenticationService = CdiUtil.bean(AuthenticationService)
         authenticationProtectionService = CdiUtil.bean(AuthenticationProtectionService)
 
-        if requestParameters.containsKey("oob:cancel"):
-            return False
-
+        session = identity.getSessionId()
```

Comment:
Im sure it is a non-issue, but this is really a session object not just an id, correct?

Reply:
Funny how that's the first thing everybody notices. I know of two  standout examples of poor naming choices in the Gluu source code. [That's one](https://github.com/GluuFederation/oxAuth/blob/master/Server/src/main/java/org/gluu/oxauth/model/common/SessionId.java).

The other is this pair of accessors [here](https://github.com/GluuFederation/fido2/blob/2b97bc5b1799bc3f4a5eb23d6e1731188406365d/server/src/main/java/org/gluu/fido2/model/auth/AuthData.java#L96-L102) which actually set or return the signature algorithm, not the key type.